storage/files.rb

```diff
@@ -46,6 +46,16 @@ def list_bucket_contents_with_prefix project_id:, bucket_name:, prefix:
   # [END list_bucket_contents_with_prefix]
 end
 
+def generate_encryption_key_base64
+  require "base64"
```

Comment:
If this isn't part of the documentation, move the require statements to the top of the file 😄